db/migrate/20210303135315_add_protected_to_locations.rb

```diff
@@ -0,0 +1,5 @@
+class AddProtectedToLocations < ActiveRecord::Migration[5.2]
```

Comment:
Should this be protected rather than protect? Maybe @harrietc52 can chip in on this one as well. Personally I think it would sound better semantically.

Reply:
Yep I would agree with changing to `protected`